playwright/mockData/orgs/KPD/people/views/AllMembers/columns.ts

```diff
@@ -2,16 +2,22 @@ import { ZetkinViewColumn } from '../../../../../../../src/types/zetkin';
 
 const AllMembersColumns: ZetkinViewColumn[] = [
     {
+        config: { field: 'first_name' },
         id: 1,
         title: 'First name',
+        type: 'person_field',
     },
     {
+        config: { field: 'last_name' },
         id: 2,
         title: 'Last name',
+        type: 'person_field',
     },
     {
+        config: { field: 'active' },
         id: 3,
         title: 'Active',
+        type: 'custom_field',
```

Comment:
I'm not sure exactly what these field's values should be, if you have a better idea @richardolsson I can correct these.

Reply:
Cool! I will send you the documentation with some configs you can copy over. 😊